index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <head>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-*********************" crossorigin="anonymous">
+      </head>
+      
+    <title>Portfolio</title>
+</head>
+<body>
+   <div class="top-navigation">
+    <nav class="nav-bar">
+        <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+   </div>
+
+   <div class="about-me">
+    <h1> I'm Massa Zoumanigui <br> glad to see you</h1>
+    <p>I’m a software developer! I can help you build a product , feature or website Look through <br> 
+        some of my work and experience! If you like what you see and have a project you need <br>
+        coded, don’t hestiate to contact me</p>
+        <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+        <button class="my-button">Click me</button>
+        <ul>
+            <li> <a href="#" target="_blank"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#" target="_blank"><i class="fab fa-git hub"></i></a></li>
+            <li><a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+        </ul>
+      
+
+
+       
+
+        
+   </div>
```

Comment:
- [ ] Good job @massazoum 👍 but as the previous reviewer said,  I think we can make this better by wrapping our content section in a main tag😉 This will make our project more semantic🤗
Also, your intro section looks a bit different from the given design and we are required in this milestone to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fbdkpd08zcETlh73-0) 😉. you can use this structure.

```
<body>
       <header>
             <a>LOGO</a>
             <nav>
                  ---- hamburger menu ----
             </nav>
       </header>
       <section>
                 ----- contents -----
       <section>
</body>
```

- [ ]  As the previous reviewer said, As you make your changes😉 take note of the font-properties, the colors used, the spacings and the background image, and ensure your project looks like the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fbdkpd08zcETlh73-0)🤗🤗🤗🤗

>The h1 text color is #172B4D
>The p text color is #344563
>font-family: 'Poppins';

- [ ] [OPTIONAL] Kindly make sure, you fixed well the background image of your section as on the   [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fbdkpd08zcETlh73-0)👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | <img width="226" alt="mobile 2 0" src="https://user-images.githubusercontent.com/57408419/219616755-cf131229-f222-4666-8d7e-1849ec48f9be.png"> | | ![mobile](https://user-images.githubusercontent.com/57408419/219697633-85154949-7d6f-4125-b293-a253d959f9a2.jpg) | |